examples/ads1115/main/main.c

```diff
@@ -3,17 +3,26 @@
 #include <freertos/task.h>
 #include <ads111x.h>
 #include <string.h>
-
-#define DEV_COUNT 2   // 2 ICs
+// Make the connections as below:
+/*
+| S. No. | ADS 1115 | ESP 32            |
+|--------|----------|-------------------|
+| 1.     | V_DD     | V_in or 5V source |
+| 2.     | GND      | GND               |
+| 3.     | SCL      | D22               |
+| 4.     | SDA      | D21               |
+| 5.     | A0       | analog input      |
+*/
+#define DEV_COUNT 1   // 1 IC
```

Comment:
This example specifically demonstrates the simultaneous use of multiple devices, which is why the default `DEV_COUNT` is 2.

Reply:
okayy